paper/paper.md

```diff
@@ -61,55 +77,89 @@ nf-core community [@ewels_nf-core_2020].
+R and Python scripting, or even manual correction in spreadsheet software.
 
-[^1]: https://en.wikipedia.org/wiki/Comma-separated_values
-[^2]: https://en.wikipedia.org/wiki/Tab-separated_values
-[^3]: https://en.wikipedia.org/wiki/OpenDocument
-[^4]: https://en.wikipedia.org/wiki/Office_Open_XML
-[^5]: https://arrow.apache.org/
-[^6]: https://parquet.apache.org/
-[^7]: https://www.hdfgroup.org/solutions/hdf5/
-
-TAXPASTA is not the first tool to attempt to standardise metagenomic profiles,
+With TAXPASTA, all of those formats can be converted into a single, standardised
+output, that, at a minimum, contains taxonomic identifiers and their relative
+abundances as integer counts. It can also be used to aggregate profiles across
+samples from the same profiler and merge them into a single, standardised table.
+Having a singular format facilitates downstream analyses and comparisons.
+TAXPASTA is not the first tool to attempt standardising metagenomic profiles,
 but it is by far the most comprehensive in terms of supported profilers and
 output formats.
 
+There exists an initiative to benchmark and compare profilers, as well as
+provide guidance on their fitness for purpose
+[@sczyrba_critical_2017; @meyer_critical_2022]. For that purpose, the
+Open-community Profiling Assessment tooL (OPAL) [@meyer_assessing_2019] was
+developed. Creating a community wide assessment faced many of the challenges
+presented here, however, the chosen solution was to mandate a single output
+format[^8] for all profilers participating in the challenge. Furthermore, OPAL
+is an integrated tool performing assessment and visualization, whereas TAXPASTA
+follows the UNIX philosophy[^9] of doing one thing well, thus allowing for
+composition with other tools.
+
+[^8]: <https://github.com/bioboxes/rfc/tree/master/data-format>
+[^9]: <https://en.wikipedia.org/wiki/Unix_philosophy#Origin>
+
 The BIOM format [@mcdonald_biological_2012] was created with a similar intention
 of standardising a storage format for microbiome analyses. However, transforming
-metagenomic profiles into that format was entirely left to the user. TAXPASTA
-is conveniently able to read profiles from a wide range of tools and can also
-produce BIOM output.
-
-The QIIME™2 _next-generation microbiome bioinformatics platform_
+metagenomic profiles into that format was entirely left up to the user. The
+format also is not easily loadable into spreadsheet software, and external
+libraries are required for loading the format into data analysis languages such
+as R. The QIIME™2 _next-generation microbiome bioinformatics platform_
 [@bolyen_reproducible_2019] also maintains internally consistent formats for
-storing and processing metagenomic data that new tools can plug into.
+storing and processing metagenomic data that new tools can plug into, however
+this suite of software is aimed mostly at 16S amplicon data [@cite], and thus
+often does have as much support or awareness in 'shotgun' sequencing [@cite]
```

Comment:
For the first citation, I think you can just cite the QIIME1 and QIIME2 papers. They don't mention metagenomics (or rather in the QIIME2 they reference 'inital support for'. For the second, it's hard to cite a non-existance of something... so I think just the two QIIME papers or maybe a link to the docs would be suffient